core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -116,7 +116,7 @@ private FileIO fileIO(Table table) {
     return table.io();
   }
 
-  private Table lazyTable() {
+  protected Table lazyTable() {
```

Comment:
Do we have to make it protected in the current implementation?

Reply:
Whoops, from old version, removed.